docs/source/deployguide/ca-deploy-topology.md

```diff
@@ -0,0 +1,58 @@
+
+#### When would I want an Intermediate CA?
+
+Intermediate CAs are optional. For added security, organizations can deploy a chain of CAs known as intermediate CAs. An intermediate CA has their root certificate issued by the root CA or another intermediate authority (that becomes the parent CA), which establishes a “chain of trust” for any certificate that is issued by any CA in the chain. Therefore, having one or more intermediate CAs allows you to protect your root of trust.  This ability to track back to the root CA not only allows the function of CAs to scale while still providing security — allowing organizations that consume certificates to use intermediate CAs with confidence — it limits the exposure of the root CA, which, if compromised, would endanger the entire chain of trust. If an intermediate CA is compromised, on the other hand, there will be a much smaller exposure. A key benefit is that after the intermediate CAs are up and running, the root CA can be effectively turned off, limiting its vulnerability even more.
+
+Another reason to include an intermediate CA would be when you have a very large organization with multiple departments and you don’t want a single CA to generate certificates for all of the departments. Intermediate CAs provide a mechanism for scoping the certificates that a CA manages to a smaller department or sub-group. Intermediate CAs are not required, but they mitigate risk and scope certificate management. This pattern incurs significant overheard if there will only be a small number of members in the organization. As an alternative to deploying multiple intermediate CAs, you can configure a CA with `affiliations` (similar to departments) instead. More on this later when we talk about `affiliations`.
+
+Also, in situations where it is acceptable for one TLS CA to be used to issue certificates to secure communications for an entire organization, it would be reasonable for intermediate CAs to use the same TLS CA as the root CA rather than having their own dedicated TLS CA.
+
+### In what order should the CAs be deployed?
+
+If a dual-headed CA is not configured with an enrollment CA and TLS CA, then the TLS CA is deployed separately, and needs to be deployed _before_ the enrollment CA in order to generate the TLS certificate for the enrollment CA. After the TLS CA is deployed, then the enrollment CA can be deployed followed by any intermediate CAs if required.
+
+## A note about configuration
+
+There are three ways to configure settings on the Fabric CA server and client. The precedence order for overriding the default settings is:
+
+1. Use the [Fabric CA server CLI commands](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/clientcli.html).
+2. Use environment variables to override configuration file settings.
+3. Modify the configuration file.
+
```

Comment:
The paragraph says to start with the configuration file, - “However, the recommendation is that before starting the server, you make the modifications to the configuration file” — , but the list says to start with the Fabric CA client. Should the order be to use the use the configuration file before things are deployed, and the fabric ca client after, with the environment variables discouraged?

Reply:
This is a great comment. 

The order aligns with the Operations Guide so I am trying to be consistent. I rewrote the entire paragraph as follows:

This order means that from a code perspective, any flags passed on a Fabric CA server CLI command will override an environment variable if it exists for the same setting, as well as the default value of the setting in the configuration file. Likewise, environment variables can be used to override settings in the configuration file. However, use of environment variables to modify configuration settings is discouraged because the changes are not persisted and can lead to problems later when they do not get set or are not set to what they should be. It's important to understand the parameters in the configuration file and their dependencies on other parameter settings in the file. Blindly overriding one setting using an environment variable could affect the functionality of another setting. Therefore, the recommendation is that before starting the server, you **make the modifications to the settings in the configuration file** to become familiar with the available settings and how they work.  Note that some configuration settings are stored in the CA database which means that after a CA is started, overriding the settings can no longer be performed by editing the configuration file or by setting environment variables. Affected parameters are noted throughout these instructions. In these cases, the modifications are required to be made by using the Fabric CA server CLI commands and have the added benefit of not requiring a server restart.

Reply:
👍